propka/atom.py

```diff
@@ -33,7 +29,13 @@
 
 
 class Atom:
-    """Atom class - contains all atom information found in the PDB file"""
+    """Atom class - contains all atom information found in the PDB file
+
+
+    .. versionchanged:: 3.4.0
+       :meth:`make_input_line` and :meth:`get_input_parameters` have been
+       removed as reading/writing PROPKA input is no longer supported.
+    """
 
     def __init__(self, line=None):
         """Initialize Atom object.
```

Comment:
`group_label`, `group_model_pka` and `group_model_pka_set` are now unused and don't need to be initialized.

Reply:
Thanks, had forgotten about those, fixed in 3ca6747 :)